parameters/reference/image-parameter/image-process-control.md

```diff
@@ -170,10 +168,13 @@ Sets the parameters passed to the library to load dynamically.
 ## ScaleDownThreshold
 Sets the threshold for the image shrinking. 
 
-**Remarks**   
-If the shorter edge size is larger than the given value, the library will calculate the required height and width of the barcode image and shrink the image to that size before localization. Otherwise, it will perform barcode localization on the original image.   
-If you have an image whose shorter edge size is larger than the given value and the barcode is a small part on it, you can set this value to be larger than the shorter edge of this image.   
-If you have an image whose shorter edge size is larger than the given value and the barcode is clear and big, you can set this value to a smaller one than the default value.
+**Remarks**
+If the shorter edge size is larger than the given value, the library will proportionally shrink the image to that size before localization. Otherwise, it will perform barcode localization on the original image.
```

Comment:
"proportionally" is not accurate. If the shorter edge size is larger than the given value, the SDK will resize the image to 25% (50% width * 50% height) for localization. @DMGithubPublisher Please confirm if it's still the case in the latest version.

Reply:
the library will shrink the image to a proper size that the target shorter edge is smaller than the given value before localization